src/irmin/object_graph.mli

```diff
@@ -40,10 +40,20 @@ module type S = sig
     min:vertex list ->
     max:vertex list ->
     unit ->
-    t Lwt.t
-  (** [closure min max pred] creates the clansitive closure of [max]
-      using the precedence relation [pred]. The closure will not
-      contain any keys before the the one specified in [min]. *)
+    (t, [ `Disconnected of vertex | `Empty_max ]) result Lwt.t
+  (** [closure min max pred] creates the transitive closure of [max] using the
+      precedence relation [pred]. Vertices that are reachable only thourgh a
```

Comment:
```suggestion
      predecessor relation [pred]. Vertices that are reachable only thourgh a
```
Not introduced by your PR, but I spotted it on the way past :slightly_smiling_face: